### DISCUSSION TITLE
HW Version 4 "7 slot" page and such gone from wiki

### DISCUSSION THREAD

**Discussion by TCorgard82** [sanni/cartreader#491] [General Chat]
I just noticed this as of today, why was this removed? I have searched and it is nowhere to be found.

**Comment by sanni** [comment]
The solder connection between the main PCB and the GBA board kept cracking all the time causing all sorts of issues.

![To-M28yE](https://user-images.githubusercontent.com/19169494/186339939-0538cbdf-2488-4dd9-bcdd-e611946af2a5.jpeg)

The seven slot HW4 was a bad build option. I torn mine apart and used the parts to build a HW5.

**Comment by TCorgard82** [comment]
Ah ok, I'll keep that in mind. Me and a buddy liked HW4 for the battery
option and 7 slot. Since looks like HW5 you have removed the battery.
